### PR TITLE
httpcaddyfile: Add certificate_pem placeholder short, add to godoc

### PR DESCRIPTION
**Pull request by gdhameeja**
Added godoc comments and CaddyFile shortcut for `certificate_pem`
placeholder

### CODE REVIEW

**Review by francislavoie — APPROVED** [review]

**Comment by francislavoie** [comment]
This is a follow-up to https://github.com/caddyserver/caddy/pull/3662 👍

File: modules/caddyhttp/app.go [hunk 1]
```diff
@@ -75,6 +75,7 @@ func init() {
 // `{http.request.tls.client.fingerprint}` | The SHA256 checksum of the client certificate
 // `{http.request.tls.client.public_key}` | The public key of the client certificate.
 // `{http.request.tls.client.public_key_sha256}` | The SHA256 checksum of the client's public key.
+// `{http.request.tls.client.certificate_pem}` | The PEM encdoed value of the certificate.
```
Comment (mholt) [inline modules/caddyhttp/app.go:78]:
```suggestion
// `{http.request.tls.client.certificate_pem}` | The PEM-encoded value of the certificate.
```

**Review by mholt — CHANGES_REQUESTED** [review]
Great, thanks so much! Just a typo :)

**Comment by gdhameeja** [comment]
> Great, thanks so much! Just a typo :)

fixed.

**Review by francislavoie — APPROVED** [review]

**Review by mholt — APPROVED** [review]

**Pull request merged**